build.gradle

```diff
@@ -92,6 +92,7 @@ dependencies {
     deobfProvided "curse.maven:fluidlogged-api-485654:4564413"
 
     deobfCompile "blusunrize:ImmersiveEngineering:0.12-92-559"
+    deobfCompile "curse.maven:engineers-decor-313866:3783239"
```

Comment:
should this be deobfProvided?

Reply:
I'd say so yeah, I was using deobfProvided for simple integrations that just add heat sources for cookware